tests/wpt/mozilla/tests/mozilla/htmlanchorelment-search-attribute.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<meta charset="utf-8">
+<html>
+<head>
+<title>HTMLAnchorElement search setter crashes</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<a>anchor</a>
```

Comment:
@jdm points to an overlap regarding the tests with #10876 and suggests:
> Test: We should create a test that exercises each of the getters and setters individually in tests/wpt/web-platform-tests/html/semantics/links/links-created-by-a-and-area-elements/.

Shall I just move it there?